packages/task/src/browser/task-definition-registry.spec.ts

```diff
@@ -105,4 +152,54 @@ describe('TaskDefinitionRegistry', () => {
             expect(defs2!.taskType).to.be.eq(definitionContributionB.taskType);
         });
     });
+
+    describe('compareTasks function', () => {
+
+        beforeEach(() => registry.register(foobarTaskFixture.def));
+
+        it('should return false if given 2 task configurations with different type', () => {
+            const res = registry.compareTasks(
+                foobarTaskFixture.conf('id_1', 'type_1'),
+                foobarTaskFixture.conf('id_2', 'type_2'),
+            );
+            expect(res).to.be.not.ok;
```

Comment:
minor: rather than `okayness`, we can simply do truthy checks.

```ts
expect(res).to.be.true;
```

or


```ts
expect(res).equal(true);
```

Reply:
Good suggestion. I'll change it in this way.